app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/address/RegistrationChangeAddressController.scala

```diff
@@ -61,8 +64,10 @@ class RegistrationChangeAddressController @Inject()(
       case _                                         => Left(Redirect(controllers.routes.StartController.start()))
     }
 
-  def updateAddress(journey: RegistrationReady, address: Address): JourneyStatus =
-    journey.copy(registrationDetails = journey.registrationDetails.copy(address = address))
+  def updateAddress(journey: RegistrationReady, address: Address)(
+    implicit hc: HeaderCarrier
+  ): EitherT[Future, Error, JourneyStatus] =
+    EitherT.pure[Future, Error](journey.copy(registrationDetails = journey.registrationDetails.copy(address = address)))
```

Comment:
is `pure` better here or `rightT`? I've been used `rightT`

Reply:
they both have the same result and it's debatable if one is better than another. `pure` let's you think of the operation in the context of a general monad whereas `rightT` is more `EitherT` specific. If you look at the implementation of `rightT` it's just an alias for `pure` - not sure why they though it was necessary to put that in